LDAR_Sim/src/file_processing/output_processing/summary_visualizations.py

```diff
@@ -21,14 +21,19 @@
 from pathlib import Path
 from typing import Tuple
 
-from matplotlib import pyplot as plt, ticker
+import matplotlib.scale as mscale
 import pandas as pd
-from constants import output_file_constants, file_name_constants
+import seaborn as sns
+
+# import scipy.stats
```

Comment:
Nit: why keep this commented out import?

Reply:
This should be removed, will resolve.